eng/tsp-core/pipelines/stages/sign-publish-tsp-cli.yml

```diff
@@ -0,0 +1,54 @@
+stages:
+  - stage: sign_tsp_cli
+    displayName: Sign tsp cli binaries
+    dependsOn: build
+
+    jobs:
+      - template: /eng/tsp-core/pipelines/jobs/cli/sign-macos.yml
+      - template: /eng/tsp-core/pipelines/jobs/cli/sign-windows.yml
+
+  - stage: verify_tsp_cli
+    displayName: Verify tsp cli
+    dependsOn: sign_tsp_cli
+    jobs:
+      - template: /eng/tsp-core/pipelines/jobs/cli/verify-tsp-cli.yml
+        parameters:
+          platform: linux-x64
+          artifactName: standalone-linux-x64
+          exePath: tsp
+
+      - template: /eng/tsp-core/pipelines/jobs/cli/verify-tsp-cli.yml
+        parameters:
+          platform: linux-arm64
+          artifactName: standalone-linux-arm64
+          exePath: tsp
```

Comment:
Just curious - why do the other OS' share an artifact name across their platforms, but not Linux? (Does it have any practical difference?)

Reply:
linux because there wasn't a signing step that merge them back